cmd/scan/scan.go

```diff
@@ -56,8 +61,10 @@ func ScanBlock(client *rpcclient.Client, blockID int64) ([]Deposit, error) {
 	for _, tx := range block.RawTx {
 		for i, addr := range tx.Vout {
 			depTx.TxHash = fmt.Sprintf("%s:%d", tx.Txid, i)
-
+			depTx.BitcoinAmount = strconv.FormatFloat(addr.Value, 'f', 8, 64)
```

Comment:
Need to use `btcutil.NewAmount` to handle this

Reply:
I can change to something like: 
```
amount, err := btcutil.NewAmount(addr.Value)
			if err != nil {
				return nil, err
			}

			depTx.BitcoinAmount = amount.String()
```
when we have string "50 BTC", is it what we need?

Reply:
No, don't need the " BTC" part, can copy their FormatFloat from here
https://github.com/btcsuite/btcutil/blob/master/amount.go#L108
which uses -8 for precision instead of 8

Also use btcutil for the next line, calculating satoshi,  e.g.:

```
satoshis := int64(btcutil.NewAmount(v.Value))  // btcutil.Amount is type int64
```